src/containers/gui.jsx

```diff
@@ -97,16 +101,18 @@ GUI.propTypes = {
 GUI.defaultProps = GUIComponent.defaultProps;
 
 const mapStateToProps = state => ({
-    activeTabIndex: state.editorTab.activeTabIndex,
-    blocksTabVisible: state.editorTab.activeTabIndex === BLOCKS_TAB_INDEX,
-    cardsVisible: state.cards.visible,
-    costumesTabVisible: state.editorTab.activeTabIndex === COSTUMES_TAB_INDEX,
-    importInfoVisible: state.modals.importInfo,
-    loadingStateVisible: state.modals.loadingProject,
-    previewInfoVisible: state.modals.previewInfo,
-    targetIsStage: state.targets.stage && state.targets.stage.id === state.targets.editingTarget,
-    soundsTabVisible: state.editorTab.activeTabIndex === SOUNDS_TAB_INDEX,
-    tipsLibraryVisible: state.modals.tipsLibrary
+    activeTabIndex: state.scratchGui.editorTab.activeTabIndex,
+    blocksTabVisible: state.scratchGui.editorTab.activeTabIndex === BLOCKS_TAB_INDEX,
+    cardsVisible: state.scratchGui.cards.visible,
+    costumesTabVisible: state.scratchGui.editorTab.activeTabIndex === COSTUMES_TAB_INDEX,
+    importInfoVisible: state.scratchGui.modals.importInfo,
+    isPlayerOnly: state.scratchGui.mode.isPlayerOnly,
+    loadingStateVisible: state.scratchGui.modals.loadingProject,
+    previewInfoVisible: state.scratchGui.modals.previewInfo,
+    targetIsStage: state.scratchGui.targets.stage &&
+    state.scratchGui.targets.stage.id === state.scratchGui.targets.editingTarget,
```

Comment:
This formatting is a little confusing (`state.scratchGui.targets.stage.id` looks like the start of a prop), I would expect something like:
```js
targetIsStage: (
    state.scratchGui.targets.stage &&
    state.scratchGui.targets.stage.id === state.scratchGui.targets.editingTarget
),

// or

targetIsStage: state.scratchGui.targets.stage &&
    state.scratchGui.targets.stage.id === state.scratchGui.targets.editingTarget,
```